rebasehelper/build_helper.py

```diff
@@ -671,3 +672,9 @@ def get_supported_tools(cls):
         :return: list of supported build tools
         """
         return build_tools.keys()
+
+    @classmethod
+    def get_default_tool(cls):
+        """Returns default build tool"""
+        default = [k for k, v in six.iteritems(build_tools) if hasattr(v, 'DEFAULT') and v.DEFAULT]
```

Comment:
@nforro you can add the DEFAULT attribute to the BuildToolBase class and set it to ```False```. Then only in the default tool class definition you can set it to ```True```. This way you won't have to use hasattr and the property of the class will be then explicitly set every time.